src/core/Microsoft.PowerToys.Settings.UI/Views/ShortcutGuidePage.xaml

```diff
@@ -92,7 +92,7 @@
             <TextBlock x:Uid="ShortcutGuide_Theme"
```

Comment:
nit: I think the changes in this file  PR this doesn't seem to have anything to do with getting unit tests running.

Reply:
sure, I reverted the changes.